### PR TITLE
edit of recipes.md

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

File: docs/docs/recipes.md [hunk 1]
```diff
@@ -192,45 +187,34 @@ since `0.7.4`).  Requires Mesos version `0.20.0` or later.
 
 ## Constraining task placement / static partitioning
```
Comment (sascala) [inline docs/docs/recipes.md]: I think this should be its own topic; it doesn't seem like a recipe to me.

File: docs/docs/recipes.md [hunk 2]
```diff
@@ -192,45 +187,34 @@ since `0.7.4`).  Requires Mesos version `0.20.0` or later.
...
+2. Configure the Mesos master: `--roles=<...other roles...>,yourrole`
 
-Configuring the Mesos Agents in question: Either use `--default_role yourrole` to assign all resources of that
-agent to your role or use the `--resources` configuration to assign individual resources to that role (such
-as a certain port range). See
-[the Mesos command line documentation](http://mesos.apache.org/documentation/latest/configuration/) for details.
+3. Configuring the Mesos agents whose resource offers you want to restrict. Either use the  `--default_role yourrole` flag to assign all that agent's resources to your role or use the `--resources` flag to assign individual resources to that role (such as a certain port range). See [the Mesos command line documentation](http://mesos.apache.org/documentation/latest/configuration/) for details.
 
-### Preventing Accidental Usage of Special Roles inside Marathon
+### Preventing Accidental Use of Special Roles inside Marathon
```
Comment (sascala) [inline docs/docs/recipes.md]: Are we saying that the directions above("Using Roles") are vulnerable to accidental use? How exactly is that?

File: docs/docs/recipes.md [hunk 3]
```diff
@@ -192,45 +187,34 @@ since `0.7.4`).  Requires Mesos version `0.20.0` or later.
...
 
-With the base setup, all Marathon applications will by default use all resources either assigned to the 
-unspecific "*" role or the role you specified with `--mesos_role`. One way to ensure that only special tasks 
-are run on these nodes is to use a separate Marathon instance for these. Another way is to configure
-Marathon such that it will ignore your special roles for apps by default and then explicitly configure the
-exceptions as shown below.
+With the base setup, all Marathon applications will by default use all resources either assigned to the unspecific "*" role or the role you specified with `--mesos_role`.
```
Comment (sascala) [inline docs/docs/recipes.md]: Is the "base setup" the setup specified above?

File: docs/docs/recipes.md (hunk 1 above)
Comment (aquamatthias) [inline docs/docs/recipes.md]: We already have a separate doc for this: https://mesosphere.github.io/marathon/docs/constraints.html where we can point to. The recipe part of it is to give you some ideas how to use it.

File: docs/docs/recipes.md (hunk 3 above)
Comment (aquamatthias) [inline docs/docs/recipes.md]: Yes. base setup == default setup == no other command line parameter supplied.

File: docs/docs/recipes.md (hunk 2 above)
Comment (aquamatthias) [inline docs/docs/recipes.md]: See below what the author meant by accidental use.  I think the wording can be improved.

**Comment by aquamatthias** [comment]
Close for now - please reopen when ready.

**Pull request closed without merging**